benchmark_runner/main/main.py

```diff
@@ -176,23 +202,23 @@ def run_clusterbuster_workload():
 @logger_time_stamp
 def main():
     """
-    The main of benchmark-runner handle Azure operations or Workload runs
+    MAIN of benchmark-runner framework
     """
     success = True
     # azure_cluster_start_stop
     if azure_cluster_stop or azure_cluster_start:
         azure_cluster_start_stop()
     # install_ocp_version
     elif install_ocp_version:
-        ibm_operations = IBMOperations(user=provision_user)
-        ibm_operations.ibm_connect()
         install_step = environment_variables_dict.get('install_step', '')
-        if install_step == 'run_ibm_ocp_installer':
-            install_ocp(ibm_operations=ibm_operations, step=install_step)
-        elif install_step == 'verify_install_complete':
-            install_ocp(ibm_operations=ibm_operations, step=install_step)
-        else:
-            logger.info(f'{ibm_operations.get_ocp_server_version().strip()} version already installed, skip OCP installation')
+        if install_step == 'run_bm_ocp_installer':
+            install_ocp_bare_metal(step=install_step)
+        elif install_step == 'run_ibm_ocp_installer':
+            install_ocp_ibm_cloud(step=install_step)
+        elif install_step == 'verify_bm_install_complete':
+            install_ocp_bare_metal(step=install_step)
+        elif install_step == 'verify_ibm_install_complete':
+            install_ocp_ibm_cloud(step=install_step)
```

Comment:
Need to handle invalid install step values.

Reply:
Fixed